.github/labeler.yml

```
@@ -0,0 +1,29 @@
+documentation:
+- any:
+  - changed-files:
+    - all-globs-to-all-files: ['docs/**', '!lib/rules/**']
+
+rule:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/rules/**']
+
+cli:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/cli.js', 'lib/options.js', 'lib/cli-engine/**']
+
+core:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/{config,eslint,linter,rule-tester,source-code}/**', 'lib/api.js']
+
+formatter:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: ['lib/cli-engine/formatters/**']
+
+github_actions:
```

Maybe to avoid being the only label with an underscore?

```suggestion
"github actions":
```

---

Actually, this label [already exists](https://github.com/eslint/eslint/pulls?q=is%3Apr+label%3Agithub_actions+is%3Aclosed), but maybe we can update it for depandabot too

---

We aren't using dependabot anymore, we're using Renovate, so we should make sure that will use the same label.